frontend/main.js

```diff
@@ -103,19 +106,45 @@ async function startApp () {
     // Since a runtime error just occured, we likely want to persist app logs to local storage now.
     sbp('appLogs/save')
   }
+
+  await sbp('gi.db/settings/load', 'CHELONIA_STATE').then(async (cheloniaState) => {
+    // TODO: PLACEHOLDER TO SIMULATE CHELONIA IN A SW
+    if (!cheloniaState) return
+    const identityContractID = await sbp('gi.db/settings/load', SETTING_CURRENT_USER)
+    if (!identityContractID) return
```

Comment:
What does loading the current userID have to do with loading the chelonia state? Could you please explain this in a comment?

Reply:
There isn't (should not be) any meaningful Chelonia state unless there's a logged in user. I.e., opening the page should not restore any state unless there's an active session

Reply:
Explained